v4l2-request-test.c

```diff
@@ -257,11 +258,34 @@ int main(int argc, char *argv[])
 	setup_config(&config);
 
 	while (1) {
-		opt = getopt(argc, argv, "v:m:d:D:s:f:P:ilqh");
+	        int option_index = 0;
+		static struct option long_options[] = {
+						       {"video_path",  required_argument, 0, 'v' },
```

Comment:
Please make the block alignment match the variable declaration with one extra tab.
Also, you should consistently add a whitespace before the first and after the last element.